src/utils/components/NetworkInterfaceModal/NetworkInterfaceModal.tsx

```diff
@@ -74,6 +83,12 @@ const NetworkInterfaceModal: FC<NetworkInterfaceModalProps> = ({
     );
   }, [nicName, networkName, interfaceModel, interfaceMACAddress, interfaceType, onSubmit]);
 
+  const vmIsRunning = getVMStatus(vm) === printableVMStatus.Running;
+  const showRestartHeader =
+    Header && (!bridgedNICHotPlugEnabled || interfaceType !== interfaceTypeTypes.BRIDGE);
```

Comment:
i would have give Header a default null value and remove the check, as Header will be null or a component

Reply:
Done!